summingbird-storm/src/main/scala/com/twitter/summingbird/storm/StormPlatform.scala

```diff
@@ -183,6 +197,12 @@ abstract class Storm(options: Map[String, Options], transformConfig: Summingbird
     }
 
     val countersForSpout: Seq[(Group, Name)] = JobCounters.getCountersForJob(jobID).getOrElse(Nil)
+    val isMergeableWithSource = getOrElse(stormDag, node, DEFAULT_FM_MERGEABLE_WITH_SOURCE).get
+    val sourceParallelism = getOrElse(stormDag, node, parOpt.getOrElse(DEFAULT_SOURCE_PARALLELISM)).parHint
+    val FMParallelism = getOrElse(stormDag, node, DEFAULT_FM_PARALLELISM).parHint
```

Comment:
We are checking on the Source because the options will be propagated to Source from FlatMapProducer unless overwritten. If the option is specified on FlatMappedProducer, we can still find it on Source and if the same option is overwritten on source, we give the priority to source. 
Will correct the variable.